test/javascripts/acceptance/dashboard-test.js.es6

```diff
@@ -86,3 +98,17 @@ QUnit.test("reports tab", async assert => {
     "filter is case insensitive"
   );
 });
+
+QUnit.test("report filters", async assert => {
+  await visit(
+    "/admin/reports/signups?end_date=2018-07-16&filters=group:88&start_date=2018-06-16"
```

Comment:
Ditto here, I thought it was moved to JSON?

Reply:
Working on it, I discovered an issue with caching, will ping you when ready for review.